### DISCUSSION TITLE
Investigation for suitable equilibrium solver for PROCESS

### DISCUSSION THREAD

**Discussion by jonmaddock** [ukaea/PROCESS#3585] [Ideas]
In GitLab by @jmorris-uk on Mar 3, 2020, 13:47

As part of 2020 EUROfusion work will investigate suitable equilibrium solvers for PROCESS

**Comment by jonmaddock** [comment]
In GitLab by @jmorris-uk on Jun 24, 2022, 11:19

closed

**Comment by mkovari** [comment]
I see that [CHEASE ](https://gitlab.epfl.ch/spc/chease) fixed-boundary axisymmetric tokamak equilibrium code is now open source(ish).

**Comment by jmorris-uk** [comment]
Not an issue or paper cut -- moved to discussion of new feature. 

The CHEASE code looks interesting and could be worth exploring. I would rather that any new fixed boundary solve in PROCESS comes in tandem with a transport solver.
